src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -36,8 +38,26 @@
  * @author Greg Korba <greg@codito.dev>
  * @author SpacePossum <possumfromspace@gmail.com>
  */
-final class FullyQualifiedStrictTypesFixer extends AbstractFixer implements ConfigurableFixerInterface
+final class FullyQualifiedStrictTypesFixer extends AbstractFixer implements ConfigurableFixerInterface, WhitespacesAwareFixerInterface
 {
+    private ImportProcessor $importProcessor;
+
+    /**
+     * @var array{
+     *     const?: array<string, class-string>,
+     *     class?: array<string, class-string>,
+     *     function?: array<string, class-string>
+     * }
+     */
+    private array $symbolsForImport = [];
+
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->importProcessor = new ImportProcessor($this->whitespacesConfig);
```

Comment:
Let's have some test with a different `WhitespacesFixerConfig` config, because I'm pretty sure this won't work. Here the config is the default one set in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.41.1/src/AbstractFixer.php#L64

When different config is used, it is set later, using `setWhitespacesConfig` from https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.41.1/src/AbstractFixer.php#L164

Reply:
Good catch 👍. It's now tested and fixed 🙂.